types.d.ts

```diff
@@ -5,28 +5,28 @@ export interface IState {
   values: object;
   search: string;
   selectBounds: object;
-  cursor: object;
+  cursor: number;
 }
 
 export interface IMethods {
-  removeItem: () => void;
-  dropDown: () => void;
+  removeItem: (event: Event, item: IItemRenderer['item'], close = false) => void;
```

Comment:
Not sure on this one. event seems to be optional in the code, but the order of the params don't make it possible

Reply:
It is not that `event` is optional, it is just might be undefined here:

https://github.com/sanusart/react-dropdown-select/blob/master/src/index.js#L238-L242

Reply:
Ok, then this type definition reflects that. However the TS compiler will insist that an event object is passed in. If that's ok with you, then please review and merge if you are happy. I've updated the event type to be the React's `SyntheticEvent` which is the basis of all the ui events in React

Reply:
Sorry, my bad - not undefuned, but `null` (here https://github.com/sanusart/react-dropdown-select/blob/master/src/index.js#L219)

Can it be defined as 
```TypeScript
removeItem: (event: Event | null, item: IItemRenderer['item'], close = false) => void;
```
you think?

Reply:
@sanusart Yes absolutely that would work. It's almost the same as making it optional. In TS, this would allow user to pass null value when they don't have an event to pass

Reply:
Usage:

```
onClick={(): void => {
  methods!.removeItem(null, item, false);
}}>
```

Reply:
@sanusart Done